finders/find_name.py

```diff
@@ -1,7 +1,30 @@
 from typing import List
-
 from finders.stops import Stop
+import re
 
 
 def find_stops_matching_name(stops: List[Stop], name: str, regex: bool = False) -> List[Stop]:
-    pass
+    found = False
+    results = list()
```

Comment:
result is only useful in the else scope, you can declare closer to where it used